src/AuthenticatedApp.jsx

```diff
@@ -1,17 +1,33 @@
+import styled from "@emotion/styled";
 import { Route, Routes } from "react-router-dom";
+import Navbar from "./components/navbar";
+import { useAuth } from "./context/auth-context";
+import ProfileRecruiter from "./pages/profilePage";
 import Search from "./pages/search";
 
+const Container = styled.div`
+  display: flex;
+  height: 100vh;
+`;
+
 function AuthenticatedApp() {
+  const { user } = useAuth();
```

Comment:
I have a question. If in the `app`, you use the values `user` and `recruiter` [link](https://github.com/sebastianterleira/get-that-job/pull/2/files#diff-d274a54187c91ba0f532df2a9e194e27ab50e988f5e4c33f5a7893918320c661R7). Isn't it necessary to use the same values here?